applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c

```diff
@@ -149,75 +149,44 @@ static int add_data(cJSON *parent, cJSON *child, const char *app_id, const char
 static int add_pvt_data(cJSON *parent, struct cloud_data_gnss *gnss)
 {
 	int err;
-
+	struct nrf_cloud_gnss_data gnss_pvt = {
+		.type = NRF_CLOUD_GNSS_TYPE_PVT,
+		.ts_ms = -1,
```

Comment:
Rather than explicitly use -1, I suggest using the aforementioned hypothetical define in nrf_cloud_codec.h